pytket/phir/cli.py

```diff
@@ -57,19 +61,24 @@ def main() -> None:
     args = parser.parse_args()
 
     for file in args.qasm_files:
-        print(f"Processing {file}")  # noqa: T201
-        c = circuit_from_qasm(file)
-        tket_opt_level = int(args.tk)
-        rc = rebase_to_qtm_machine(c, args.machine, tket_opt_level)
```

Comment:
Where are we doing this rebasing now?

Reply:
The gate set rebase happens in `pytket_to_phir`